scripts/skmem.pl

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/perl
+#
+# Copyright (C) 2021 Tempesta Technologies, Inc.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License,
+# or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program; if not, write to the Free Software Foundation, Inc., 59
+# Temple Place - Suite 330, Boston, MA 02111-1307, USA.
+use 5.16.0;
+use strict;
+use warnings;
+
+my ($rmem, $rcv_buf, $wmem, $snd_buf, $fwd_alloc) = (0, 0, 0, 0, 0);
+my ($wmem_q, $ropt_mem, $back_log, $sock_drop) = (0, 0, 0, 0);
+
+# See linux/net/ipv4/inet_diag.c for the fields reporting.
+my @ss = `ss -tm '( dport = :https or sport = :https )' | grep skmem`;
```

Comment:
I use a custom-built kernel with a config that leans towards minimal working configuration (to speed-up rebuild times). It lacked at least CONFIG_INET_DIAG, CONFIG_INET_TCP_DIAG, and CONFIG_NETLINK_DIAG, so `ss` used `/proc/net/tcp` rather than getting information over a netlink socket.

With rebuilt kernel that now have mentioned config options enabled, `ss` shows `skmem:` lines in the VM too.